binarytree/layout.py

```diff
@@ -0,0 +1,113 @@
+""" Module containing layout related algorithms."""
+
+
+def _get_coords(values):
+    """Generate nodes and edges relative coordinates lists
```

Comment:
Please change to `Generate the coordinates used for rendering the nodes and edges.`